web/app/js/components/Sidebar.jsx

```diff
@@ -108,32 +139,23 @@ class Sidebar extends React.Component {
 
             <Menu.Item className="sidebar-menu-item" key="/namespaces">
               <ConduitLink to="/namespaces">
-                <Icon>N</Icon>
+                <Icon type="dashboard" />
                 <span>Namespaces</span>
               </ConduitLink>
             </Menu.Item>
 
-            <Menu.Item className="sidebar-menu-item" key="/deployments">
-              <ConduitLink to="/deployments">
-                <Icon>D</Icon>
-                <span>Deployments</span>
-              </ConduitLink>
-            </Menu.Item>
-
-            <Menu.Item className="sidebar-menu-item" key="/replicationcontrollers">
-              <ConduitLink to="/replicationcontrollers">
-                <Icon>R</Icon>
-                <span>Replication Controllers</span>
-              </ConduitLink>
-            </Menu.Item>
-
-            <Menu.Item className="sidebar-menu-item" key="/pods">
-              <ConduitLink to="/pods">
-                <Icon>P</Icon>
-                <span>Pods</span>
-              </ConduitLink>
-            </Menu.Item>
-
+            {
+              _.map(this.state.namespaces, ns => {
+                return (
+                  <Menu.Item className="sidebar-submenu-item" key={ns}>
+                    <ConduitLink to={`/namespaces/${ns}`}>
+                      <Icon>{_.take(ns, 2)}</Icon>
+                      <span>{ns} {this.state.collapsed ? "namespace" : ""}</span>
```

Comment:
how about only showing the namespace names when the sidebar is expanded, rather than abbreviation + name? maybe leave an indent so it communicates it's a subitem of `Namespaces`?